setup-plugdev-perm.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Setup plugdev permissions for the device
```

Comment:
I know you commented in this PR what this is doing, but for later when that's long forgotten, it'd be useful to have that documented in here too.